src/element.ts

```diff
@@ -491,4 +483,95 @@ export abstract class Element {
     const fontInfo = this.fontInfo;
     this.setFont(fontInfo.family, fontInfo.size, weight, fontInfo.style);
   }
+
+  /** Get element width. */
+  getWidth(): number {
+    return this.width;
+  }
+
+  /** Set element width. */
+  setWidth(width: number): this {
+    this.width = width;
+    return this;
+  }
+
+  /** Shift element down `y` pixels. Negative values shift up. */
+  setYShift(y: number): this {
+    this.yShift = y;
+    return this;
+  }
+
+  /** Get shift element `y` */
+  getYShift(): number {
+    return this.yShift;
+  }
+
+  /**
+   * Set shift element right `x` pixels. Negative values shift left.
+   */
+  setXShift(x: number): this {
```

Comment:
Same as with yShift, I think maybe it should be `dX` or `xShift`.

Reply:
done